### DISCUSSION TITLE
Should we create issues from discussions comparing to TAXSIM that identify a bug in PE?

### DISCUSSION THREAD

**Discussion by MaxGhenis** [PolicyEngine/policyengine-us#3941] [Ideas]
Currently our typical flow is for @martinholmer to create a discussion thread when we have a discrepancy with TAXSIM, at which point, if it results from a PE bug, @PavelMakarchuk creates an issue to address it, cross-referencing the issue to the discussion and closing the discussion when it's resolved. An alternative flow could be turning the discussion into an issue.

The advantage of the current flow is that we can continue the discussion if the identified issue doesn't wholly resolve it. On the other hand, turning discussions into issues is more streamlined, requiring one fewer "item" and removing the need to resolve the discussion separately after closing the issue.

Which should we adopt?

**Comment by martinholmer** [comment]
Which ever approach makes more sense to @MaxGhenis and @PavelMakarchuk.
Just let me know and I'll try to fit into the workflow.

**Reply by PavelMakarchuk** [reply]
I believe turning the discussion into an issue would be best as we usually re-run integration tests after fixing issues.

**Reply by MaxGhenis** [reply]
Sounds good. Martin I don't think this requires any change on your end then.

**Reply by PavelMakarchuk** [reply]
I will make a better effort of tagging the issue in the discussion as we open them

**Reply by MaxGhenis** [reply]
You won't need to tag issues in discussions, just `Create issue from discussion` and then retitle it to target the issue a PR would address.
